### PR TITLE
Update README.md, add LICENSE

### PR DESCRIPTION
**Pull request by tranchitella**
Changelog: none

Signed-off-by: Fabio Tranchitella <fabio.tranchitella@northern.teech>

### CODE REVIEW

**Review by kacf — CHANGES_REQUESTED** [review]
Looks good, but can you also add the `.gitlab-ci-check-license.yml` test in `.gitlab-ci.yml`?

File: README.md [hunk 1]
```diff
@@ -1,15 +1,18 @@
...
-==============================================
+Mender: remote shell access add-on
+==================================
 
 Mender is an open source over-the-air (OTA) software updater for embedded Linux
 devices. Mender comprises a client running at the embedded device, as well as
 a server that manages deployments across many devices.
 
+This repository contains the remote shell access add-on. It enhances the
+[Mender client](https://coveralls.io/repos/github/mendersoftware/mender), allowing
```
Comment (merlin-northern) [inline README.md:13]: maybe lets link to the https://github.com/mendersoftware/mender ?

File: README.md [hunk 2]
```diff
@@ -31,7 +34,7 @@ documentation](https://github.com/mendersoftware/mender/blob/master/CONTRIBUTING
 ## License
 
 Mender is licensed under the Apache License, Version 2.0. See
-[LICENSE](https://github.com/mendersoftware/inventory/blob/master/LICENSE) for the
+[LICENSE](https://github.com/mendersoftware/mender-shell/blob/master/LICENSE) for the
```
Comment (merlin-northern) [inline README.md:37]: thanks for correcting that!

**Review by merlin-northern — COMMENTED** [review]
thanks for the fixes.

File: README.md (hunk 1 above)
Comment (tranchitella) [reply README.md]: fixed, sorry, that's what I inteded.

**Comment by tranchitella** [comment]
@kacf I've added the LICENSE checksums and the golang test, as we do in the backend services.

**Review by kacf — DISMISSED** [review]

**Review by lluiscampos — DISMISSED** [review]
:+1: 

For the license test, we could have included the template instead (what @kacf suggested) as we are going to revisit all backend repositories soon to amend it (see [QA-238](https://tracker.mender.io/browse/QA-238)).

Anyway, as-is it is as good as the rest of repos :+1:

**Comment by tranchitella** [comment]
@kacf @lluiscampos reworked to use the template, one repo less to convert :)

File: .gitlab-ci.yml [hunk 3]
```diff
@@ -12,6 +12,8 @@ include:
     file: '.gitlab-ci-check-golang-static.yml'
   - project: 'Northern.tech/Mender/mendertesting'
     file: '.gitlab-ci-check-golang-unittests.yml'
+  - project: 'Northern.tech/Mender/mendertesting'
+    file: '.gitlab-ci-check-license.yml'
```
Comment (lluiscampos) [inline .gitlab-ci.yml:16]: Not very clear (I'll update JIRA description now), but we are also [.gitlab-ci-check-commits.yml](https://github.com/mendersoftware/mendertesting/blob/master/.gitlab-ci-check-commits.yml). The esoteric `checkMenderCompliance` from the gocode calls both scripts.

**Review by lluiscampos — CHANGES_REQUESTED** [review]
> @kacf @lluiscampos reworked to use the template, one repo less to convert :)

Thanks! Missing one part of it, but I appreciate the initiative :)

File: go.mod [hunk 4]
```diff
@@ -6,5 +6,6 @@ replace github.com/urfave/cli/v2 => github.com/mendersoftware/cli/v2 v2.1.1-mini
 
 require (
 	github.com/sirupsen/logrus v1.7.0
+	github.com/stretchr/testify v1.5.1 // indirect
```
Comment (kacf) [inline go.mod:9]: Is this needed now?

Comment (tranchitella) [reply go.mod:9]: I believe it comes from logrus

File: .gitlab-ci.yml (hunk 3 above)
Comment (tranchitella) [reply .gitlab-ci.yml:18]: added that template, too

**Review by lluiscampos — APPROVED** [review]

**Pull request merged**